source/JsMaterialX/README.md

```diff
@@ -31,22 +31,31 @@ The Emscripten toolchain is documented [here](https://emscripten.org/docs/buildi
 In the root of directory of this repository run the following:
 
 #### Docker
-It is recommended to build the project with [docker](https://docs.docker.com/) here are the required steps:
+It is recommended to build the project with Docker, here are the required steps:
+
+  1. Download and install [Docker](https://docs.docker.com/) as instructed on the documentation.
+
+     For Windows make sure to use Linux containers and that File Sharing is set up to allow local directories on Windows to be shared with Linux containers. 
 
-  1. For Windows make sure to use Linux containers and that File Sharing is set up to allow local directories on Windows to be shared with Linux containers. 
-  
      For example, if the path to MaterialX is ```"c:\git\MaterialXrepo"``` then the ```"c"``` drive should be set. (See https://docs.docker.com/docker-for-windows/#file-sharing for more details)
 
+     If you work on WSL (Windows Subsystem for Linux), install [Docker](https://docs.docker.com/docker-for-windows/install/) for Windows.
+
   2. Get the `emscripten` docker image
      ```sh
-     docker run -dit --name emscripten -v {path_to_MaterialX}:/src trzeci/emscripten:1.39.7-upstream bash
+     docker run -p 8080:8080 -dit --name emscripten -v {path_to_MaterialX}:/src trzeci/emscripten:1.39.7-upstream bash
      ```
+     For WSL, make sure to run the command on a Windows terminal and also to adjust the path (e.g, `"C:\Users\{windows_username}\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_{local_code}\LocalState\rootfs\{WSL_path_to_MaterialX}"`).
+
```

Comment:
Can't comment on the line directly, but can you split the configuration and building into two steps. Like it is done in the CMake section.